tests/foreman/maintain/test_clone.py

```diff
@@ -0,0 +1,163 @@
+"""Tests for satellite-maintain clone
+
+:Requirement: foreman-maintain
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: SatelliteClone
+
+:Assignee: lpramuk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+import requests
+
+from robottelo import constants
+from robottelo.config import settings
+
+USER = settings.subscription.rhn_username
+PASSWORD = settings.subscription.rhn_password
+SSH_PASS = settings.server.ssh_password
+POOL = settings.subscription.rhn_poolid
+BACKUP_DIR = '/var/backup'
+
+
+@pytest.mark.parametrize(
+    "sat_ready_rhel", [7, 8] if settings.server.version.rhel_version < 8 else [8], indirect=True
+)
```

Comment:
Given how we execute the tests, I'd not parametrize it regardless.

Otherwise the sat-6.11-rhel7-SatelliteClone pipeline will run the test against a rhel7 sat, and the sat-6.11-rhel8-SatelliteClone pipeline will run it against both - repeating the test from the first pipeline.

The fixture should just use the OS version of satellite that's in settings. Replicate the target satellite's base OS. The separate pipelines will cover the two base OS 'parameters'.

Reply:
I am not sure that I follow your comment 100%.
The support matrix is the following:
```
Satellite 6.11 on RHEL 7 ---clone---> Satellite 6.11 on RHEL 7
Satellite 6.11 on RHEL 7 ---clone---> Satellite 6.11 on RHEL 8
Satellite 6.11 on RHEL 8 ---clone---> Satellite 6.11 on RHEL 8
```
So the `sat-6.11-rhel7-SatelliteClone` pipeline should also test that we can clone from RHEL 7 to RHEL 8.

Reply:
ack, good catch @ogajduse, still needs the parametrization in `6.11.z` branch, just not in the master branch.